examples/reference-entity-model/entities/Customer.entity.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
I believe the current "api/entities/<entity_name>" endpoint would require all sub entities to be in separate entity files as well. So, these entities will not be rendered as expected in the mapping UI screen